fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindExpression.java

```diff
@@ -697,6 +702,107 @@ private Plan bindFilter(MatchingContext<LogicalFilter<Plan>> ctx) {
         return new LogicalFilter<>(boundConjuncts.build(), filter.child());
     }
 
+    private Plan bindQualify(MatchingContext<LogicalQualify<Plan>> ctx) {
+        LogicalQualify<Plan> qualify = ctx.root;
+        CascadesContext cascadesContext = ctx.cascadesContext;
+        ImmutableSet.Builder<Expression> boundConjuncts = ImmutableSet.builderWithExpectedSize(
+                qualify.getConjuncts().size());
+        qualify.accept(new DefaultPlanVisitor<Void, Void>() {
+            @Override
+            public Void visitLogicalProject(LogicalProject<? extends Plan> project, Void context) {
```

Comment:
we could use pattern to distinguish them? `logicalQualify(logicalProject())` and `logicalQualify(logicalAggregate())` ?